src/ctia/schemas/core.clj

```diff
@@ -16,82 +16,169 @@
              [verdict :as vs]
              [vocabularies :as vocs]]
             [flanders
-             [schema :as fs]
+             [schema :as f-schema]
+             [spec :as f-spec]
              [utils :as fu]]
-            [schema.core :refer [defschema Str Bool]]))
+            [schema.core :refer [Str Bool] :as sc]))
+
+(defmacro defschema [name-sym ddl spec-kw-ns]
```

Comment:
I think the main benefit is that looks cleaner.  It if was a fn, it would have to quote the name-sym in the caller.